lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandler.java

```diff
@@ -302,6 +314,44 @@ private JWEObject signEncryptJar(
         return AuthorizationRequestHelper.createJweObject(rsaEncrypter, signedJWT);
     }
 
+    private EvidenceRequest getEvidenceRequestForF2F(List<SignedJWT> credentials)
+            throws UnknownEvidenceTypeException, ParseException {
+        Gpg45Scores gpg45Scores = gpg45ProfileEvaluator.buildScore(credentials);
+        List<List<Gpg45Scores.Evidence>> requiredEvidences =
+                gpg45ProfileEvaluator.calculateEvidencesRequiredToMeetAProfile(
+                        gpg45Scores, ACCEPTED_PROFILES);
+
+        if (requiredEvidences == null) {
+            return null;
+        }
```

Comment:
When is this null?

Reply:
Ah, this was a mistake. You're right it can't be null